.helm/ecamp3/templates/hook_db_migrate.yaml

```
@@ -21,7 +21,7 @@ spec:
       containers:
         - name: db-migrate-job
           {{ template "api.phpContainer" . }}
-          command: ["migrate-database"]
+          command: ["migrate-database -e dev"]
```

Hier könntest du auch die Go-Templates verwenden um `.Values.php.appEnv` oder einen anderen value einzufügen:
```suggestion
          command: ["migrate-database -e {{ .Values.php.appEnv }}"]
```
Der Helm Chart sollte für die Allgemeinheit nutzbar sein und auf einem beliebigen Kubernetes-Cluster mit minimaler Konfiguration eine (produktive) Instanz produzieren können. Einzelne Values für ein Deployment überschreiben können wir in https://github.com/ecamp/ecamp3/blob/devel/.github/workflows/continuous-deployment.yml#L203..L221

Langfristig fände ich es cool, wenn mindestens dev.ecamp3.ch weiterhin im appEnv prod laufen könnte, oder sonst dann irgendwann mal ein Staging Environment. Im Optimalfall wäre also das Migrations-Environment sogar ein eigener Value unabhängig vom appEnv.

---

Hat so oder so nicht geklappt, da die "dev" konfiguration web_profiler.yaml laden möchte, was es gar nicht gibt.
Hab jetzt das directory als environemnt variable durchgeschlauft, jetzt können wir sogar je nach umgebung unterschiedliche dumps verwenden.